### PR TITLE
Allow requirejs version 2.2

### PR DESCRIPTION
**Pull request by jbrunel**
PR for issue #41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "peerDependencies": {
     "karma": ">=0.9",
-    "requirejs": "~2.1"
+    "requirejs": "2.1.x | 2.2.x"
```
Comment (delfrrr) [inline package.json]: May be just
```js
"requirejs": "^2.1.0"
```
Otherwise, we will have the same issue after requirejs 2.3 release

Comment (dignifiedquire) [inline package.json]: Yes, please let's do that

**Comment by dignifiedquire** [comment]
Thanks, can you update the commit message to follow our [convention](http://karma-runner.github.io/0.13/dev/git-commit-msg.html) please?

**Comment by dignifiedquire** [comment]
Thanks fully fixed in https://github.com/karma-runner/karma-requirejs/commit/c588765c9925ea287d6cc87ff8698a1fbbfd5ac8

**Pull request closed without merging**